nextcloudappstore/core/templates/app/list.html

```diff
@@ -8,15 +8,26 @@
 {% endblock %}
 
 {% block apps %}
+
+{% if search_query %}
+    <div class="col-lg-6">
+        {% if search_query and current_category %}
+            <h2 class="search-results-title">{% blocktrans with cat=current_category q=search_query%}Search results for "{{ q }}" in {{ cat }}{% endblocktrans %}</h2>
```

Comment:
probably better to spell out query and category variables inside the translation so their meaning is obvious to translators